src/MonologCreator/Factory.php

```diff
@@ -130,6 +130,13 @@ public function createProcessors(array $loggerConfig)
                 $processors[] = $webProcessor;
             } elseif ('requestId' === $processor) {
                 $processors[] = new Processor\RequestId();
+            } elseif ('extraField') {
+                $extraFields = null;
+                if (is_array($loggerConfig['extraFields'])) {
+                    $extraFields = $loggerConfig['extraFields'];
+                }
+
+                $processors[] = new Processor\ExtraFieldProcessor($extraFields);
```

Comment:
So from the code it could be that this line gets called with `$extraFields` being `null`. Is this an intended behavior or should this line be in the above `if` block?

Reply:
This is intended behavior.
ExtraFieldProcessor handles being called with `null` itself to better work with possibly different implementations of the Factory class.
The processor wouldn't be configured for the logger if this line was in the `if` block, leading to inconsistent test results.

Reply:
Understood.